lib/webdriver.js

```diff
@@ -108,9 +108,12 @@ Webdriver.prototype._init = function() {
     // retrieving session
     try{
       var jsonData = JSON.parse(data);
-      if( jsonData.status === 0 ){
+
+      _this.sessionID = jsonData.value.sessionId;
```

Comment:
what if `jsonData.value` is undefined? we probably want a check here to make sure things don't blow up. something like:

```js
_this.sessionId = jsonData.value ? jsonData.value.sessionId : false;
```

this is safer, and still dumps into the conditional below if we didn't get an appropriate value from it.